.gitdocs.js

```diff
@@ -7,5 +7,20 @@ var config = require('./docs/.gitdocs.json');
 
 module.exports = new Promise((resolve, reject) => {
   config.root = 'docs/';
+  setupHost(config);
   resolve(config);
 });
+
+function fetchAppDomain() {
+  const { execSync } = require('child_process');
+
+  return execSync('rake app_domain').toString().replace(/\s/g, '');
```

Comment:
I don't think we need a rake tasks for this, we can just use the Rails runner:

```sh
$ rails runner 'puts ApplicationConfig.app_domain_no_port'
localhost
```
```suggestion
  return execSync('rails runner 'puts ApplicationConfig.app_domain_no_port'').toString().replace(/\s/g, '');
```

Reply:
Thanks for the suggestion and providing the fix! Always a pleasure working with you.